docs/datastore/alloydb.md

```diff
@@ -102,7 +107,8 @@ connect to your VPC. You should only need to do this once per VPC (per project).
         --cpu-count=8 \
         --region=$REGION \
         --cluster=$CLUSTER \
-        --project=$PROJECT_ID
+        --project=$PROJECT_ID \
+        --ssl-mode=ALLOW_UNENCRYPTED_AND_ENCRYPTED
```

Comment:
Let's try to remember to remove this once we have proper cloud sql and alloydb connectors being used.